From b3a65a0b22bd9ad314f9cf8a9ea8a9a28ac1877c Mon Sep 17 00:00:00 2001
From: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
Date: Thu, 26 Mar 2020 14:19:45 +0200
Subject: [PATCH 71/89] platform/x86: surface3_power: Use dev_err() instead of
 pr_err()

We have device and we may use it to print messages.

Signed-off-by: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
---
 drivers/platform/x86/surface3_power.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/platform/x86/surface3_power.c b/drivers/platform/x86/surface3_power.c
index 9cdae35c9802..3eb1b48c562e 100644
--- a/drivers/platform/x86/surface3_power.c
+++ b/drivers/platform/x86/surface3_power.c
@@ -432,7 +432,7 @@ mshw0011_space_handler(u32 function, acpi_physical_address command,
 		ret = mshw0011_bst(cdata, &gsb->bst);
 		break;
 	default:
-		pr_info("command(0x%02x) is not supported.\n", gsb->cmd.arg1);
+		dev_info(&cdata->bat0->dev, "command(0x%02x) is not supported.\n", gsb->cmd.arg1);
 		ret = AE_BAD_PARAMETER;
 		goto err;
 	}
-- 
2.30.2

